lib/percy/config.rb

```diff
@@ -42,7 +42,7 @@ def debug
     end
 
     def repo
```

Comment:
Just like you did elsewhere we should kill this method entirely — any reference to `repo` is super old and we should be ok to remove nowadays (and this method is almost guaranteed to only have been used internally, or set via env vars). It's not even needed to keep this for backwards compatibility really.